include/librealsense2/h/rs_sensor.h

```diff
@@ -667,6 +667,24 @@ void rs2_set_motion_device_intrinsics(const rs2_sensor* sensor, const rs2_stream
 */
 float rs2_get_max_usable_depth_range(rs2_sensor const * sensor, rs2_error** error);
 
+/**
+* rs2_get_saftey_preset
+* \param[in]   sensor         Safety sensor
+* \param[in]   index          Index to read from
+* \param[out]  safetry_preset
```

Comment:
`rs2_safety_preset`

Reply:
Even change to the real name `sp` (or update the name), just match

Reply:
done